src/battle-log.ts

```diff
@@ -454,7 +454,7 @@ class BattleLog {
 	static usernameColor(name: ID) {
 		if (this.colorCache[name]) return this.colorCache[name];
 		let hash;
-		if (window.Config?.customcolors?.[name]) {
+		if (Config.customcolors?.[name]) {
```

Comment:
```suggestion
		if (Config.customcolors[name]) {
```

Let's make this required inside the file.